src/main/java/org/alliancegenome/curation_api/services/helpers/diseaseAnnotations/DiseaseAnnotationCurie.java

```diff
@@ -24,7 +24,13 @@ public String getExperimentalConditionCurie(ExperimentalConditionDTO dto) {
 
     public String getEvidenceCurie(EvidenceDTO dto) {
         CurieGenerator curie = new CurieGenerator();
-        curie.add(dto.getPublication().getCrossReference().getCurie());
+
+        if(dto.getPublication().getCrossReference() != null){
+            curie.add(dto.getPublication().getCrossReference().getCurie());
+        }else{
+            curie.add(dto.getPublication().getPublicationId());
+        }
+
```

Comment:
@cmpich the publication -> crossreference is optional. Not all mods have those so the logic is use the ModId if it exists otherwise use the PubMedId.